### PR TITLE
changed a phrase

### PR DESCRIPTION
**Pull request by ghost**
💯 fixed issue.

### CODE REVIEW

**Comment by ghost** [comment]
✅ beginning github work flow.

**Comment by QuantumShadow87** [comment]
> 100 fixed issue.

**Comment by QuantumShadow87** [comment]
A1

File: index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 <!-- Feel free to change this text here -->
 <p>
-  Fork me? Fork you, @octocat!
+ I'm victor0x16, I want to help you!
```
Comment (QuantumShadow87) [inline index.html:16]: index.html

**Review by QuantumShadow87 — COMMENTED** [review]
A1 let's go

**Pull request closed without merging**